composer.lock

```diff
@@ -448,7 +447,7 @@
             "keywords": [
                 "database"
             ],
-            "time": "2015-03-30 12:14:13"
+            "time": "2015-03-30T12:14:13+00:00"
```

Comment:
This change in the date format is due to the upgrade of Composer to the 1.3 version.